src/core/components/courses/TrainingContractContainer.vue

```diff
@@ -306,6 +306,8 @@ export default {
 
     created();
 
+    context.expose({ refreshTrainingContracts });
```

Comment:
On a déjà utilise ce processus ? Est-ce que tu peux nous indiquer ce que tu as regarde sur ce sujet et ce qui t'a fait choisir cette methode ?

Reply:
non on ne l'a pas utilisé pour le moment, j'ai suivi ce tuto https://stackoverflow.com/questions/40957008/how-to-access-to-a-child-method-from-the-parent-in-vue-js
c'est ce qui m'a été proposé en cherchant comment déclencher une fonction dans un composant depuis un composant parent

Reply:
Est-ce que tu as regarde sinon pour mettre refreshTrainingContract dans ProfileOrga et d'envoyer depuis TrainingContractContainer un emit(refresh-training-contract) quand il y a besoin ? J'ai l'impression qu'on fait ca parfois (ex dans CourseBillingCard)

Reply:
done